server/config.ts

```diff
@@ -4,7 +4,11 @@ const production = process.env.NODE_ENV === 'production'
 
 function get<T>(name: string, fallback: T, options = { requireInProduction: false }): T | string {
   if (process.env[name]) {
-    return process.env[name]
+    if (production) {
+      return process.env[name]
+    }
+
+    return process.env[name].replace(/\\/g, '')
```

Comment:
Is this the fix for `.env` issues you had?

Reply:
Yep, I left production as is, although maybe I need to work out when we're running locally?

Reply:
👍 perhaps you could just have something else in the `.env` file, like `MESSY_TOKEN=true`!